GCC 6.2.0: Compile Error

Compiling with gcc 6.2.0 leads to a compile error in [`DCHelper.hpp` line 150](https://github.com/ComputationalRadiationPhysics/libSplash/blob/v1.5.0/src/include/splash/core/DCHelper.hpp#L150):
```
In file included from src/SerialDataCollector.cpp:38:0:
src/include/splash/core/DCHelper.hpp: In static member function static void splash::DCHelper::getOptimalChunkDims(const hsize_t*, uint32_t, size_t, hsize_t*):
src/include/splash/core/DCHelper.hpp:150:85: error: call of overloaded abs(size_t) is ambiguous
                 size_t chunk_diff = abs(target_chunk_size - (current_chunk_size * 2));
```

--- a/src/include/splash/core/DCHelper.hpp
+++ b/src/include/splash/core/DCHelper.hpp
@@ -24,8 +24,6 @@
 #define DCHELPER_H
 
 #include <map>
-#include <cmath>
-#include <cstdlib>
 #include <sstream>
 #include <iostream>
 #include <hdf5.h>
@@ -147,7 +145,7 @@ namespace splash
             while (current_chunk_size < target_chunk_size)
             {
                 // test if increasing chunk size optimizes towards target chunk size
-                size_t chunk_diff = abs(target_chunk_size - (current_chunk_size * 2));
+                size_t chunk_diff = target_chunk_size - (current_chunk_size * 2u);
                 if (chunk_diff >= last_chunk_diff)
                     break;
 

